msu/systems/keybinds/key_static.nut

```diff
@@ -173,4 +192,212 @@
 		"96" : "shift",
 		"97" : "alt"
 	}
+	ReverseMouseMapSQ = {
+		"leftclick" : "1",
+		"rightclick" : "2",
+	},
+	ReverseKeyMapSQ = {
```

Comment:
my only thought is whether it might be worth creating these programmatically from the other tables so that the data only isn't duplicated and we don't have to remember to maintain both separately.

Reply:
I doubt these will change anytime soon.

Reply:
This is fair I guess, just bothers me conceptually.

Reply:
Technically it's PERF

Reply:
I mean sure, but I'm almost always in favor of decreased code duplication over increased performance, otherwise I wouldn't have thought up the callSkillsFunction stuff and just written it all out each time.

Reply:
Did it for SQ, not doable for JS without complications.